manifest.versions.json

```
@@ -209,11 +209,11 @@
     "powershell|9.0|Windows|x64|sha": "44a7a9af838aa6138517024771903464ab6eabf46b1eadd6b0d7e4c2d1bbbda375c21af3e7d60a6805b0c03d2a45bd17c96f688049f03fab3877d4c73e20c278",
 
     "powershell|10.0|build-version": "$(powershell|9.0|build-version)",
```

It looks like the build version wasn't updated.

```suggestion
    "powershell|10.0|build-version": "7.6.0-preview.3",
```